etc/profile-a-l/lbry-viewer.profile

```diff
@@ -0,0 +1,21 @@
+# Firejail profile for lbry-viewer
+# Description:CLI viewer for searching and playing videos from LBRY, with the Librarian frontend
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include lbry-viewer.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/lbry-viewer
+noblacklist ${HOME}/.config/lbry-viewer
+
+mkdir ${HOME}/.config/lbry-viewer
+mkdir ${HOME}/.cache/lbry-viewer
+whitelist ${HOME}/.cache/lbry-viewer
+whitelist ${HOME}/.config/lbry-viewer
+
+private-bin gtk-lbry-viewer,lbry-viewer
```

Comment:
```suggestion
private-bin lbry-viewer
```

Note that `private-bin` accumulates, so it might make more sense to put
`private-bin gtk-lbry-viewer` on gtk-lbry-viewer.profile instead.